index.html

```diff
@@ -384,9 +384,9 @@ <h2 style="color:white" class="section-heading">Contact Us</h2>
                     <span class="error" id="error_message"></span>
                   </div>
                   <div class="form_group">
-                    <div class="row">
+                    <div class="row" style="margin: 0 auto; width: 656px; text-align: center;" >
                       <span id="success" class="col-xs-4"></span>
```

Comment:
So essentially, just make this success message appear after the send button in a separate row. That way both of them can use the `center-block` class.